### PR TITLE
Added upload service

### PR DESCRIPTION
**Pull request by Omermigdal**
Same here regarding Swagger UI .. not allowed

### CODE REVIEW

File: source/FilesServer/app/service.py [hunk 1]
```diff
@@ -15,3 +16,11 @@ async def s3_download(file_name: str) -> str:
...
     )
     return presigned_url
+
+
+async def s3_upload(file: UploadFile = File(...)) -> None:
+    settings = get_settings()
+    s3_client = get_s3_client()
+    file_name = file.filename
+    s3_client.upload_fileobj(file.file, settings.aws_bucket_name, file_name)
+    return
```
Comment (tal-alon) [inline source/FilesServer/app/service.py]: remove redundant return statement

File: source/FilesServer/app/service.py [hunk 2]
```diff
@@ -15,3 +16,11 @@ async def s3_download(file_name: str) -> str:
         ExpiresIn=settings.presigned_url_expiration,
     )
     return presigned_url
+
+
+async def s3_upload(file: UploadFile = File(...)) -> None:
```
Comment (tal-alon) [inline source/FilesServer/app/service.py]: i think we can simplify the type annotation to:
...(file: File) -> ....

File: source/FilesServer/app/endpoints.py [hunk 3]
```diff
@@ -15,6 +15,10 @@ async def download(file_name: str) -> RedirectResponse:
     return RedirectResponse(url=presigned_url)
 
 
-@router.post("/upload")
-async def upload(file: UploadFile | None = None):
-    return await service.s3_upload(file)
+@router.post("/upload", status_code=status.HTTP_200_OK)
+async def upload(file: UploadFile) -> None:
+    if not file:
```
Comment (tal-alon) [inline source/FilesServer/app/endpoints.py]: has you checked if fastapi+pydantic are checking this for us?
file is annotated as "UploadFile" not "UploadFile | None" so i think it shouldn't be None-able. if so, this if statement is redundant

**Review by tal-alon — CHANGES_REQUESTED** [review]

File: source/FilesServer/app/service.py (hunk 2 above)
Comment (Omermigdal) [reply source/FilesServer/app/service.py]: fixed

File: source/FilesServer/app/endpoints.py (hunk 3 above)
Comment (Omermigdal) [reply source/FilesServer/app/endpoints.py]: yup

**Review by tal-alon — APPROVED** [review]

**Pull request merged**